kivy_ios/toolchain.py

```diff
@@ -85,6 +87,16 @@ def remove_junk(d):
                 unlink(join(root, fn))
 
 
+def https_download(url, filepath):
+    http = urllib3.PoolManager(cert_reqs="CERT_REQUIRED", ca_certs=certifi.where())
+    with http.request("GET", url, preload_content=False) as resp, open(
```

Comment:
That call doesn't check about errors.

Example:
If a 404 is returned, it just downloads the 404 page and save that into the provided path.